controllers/dspipeline_controller.go

```diff
@@ -397,7 +409,16 @@ func (r *DSPAReconciler) handleReadyCondition(
 
 }
 
-func (r *DSPAReconciler) GenerateStatus(ctx context.Context, dspa *dspav1alpha1.DataSciencePipelinesApplication) ([]metav1.Condition, error) {
+func (r *DSPAReconciler) GenerateStatus(ctx context.Context, dspa *dspav1alpha1.DataSciencePipelinesApplication, params *DSPAParams) ([]metav1.Condition, error) {
+	databaseAvailable := r.buildCondition(config.DatabaseAvailable, dspa, config.DatabaseAvailable)
+	if r.isDatabaseAccessible(ctx, dspa, params) {
+		databaseAvailable.Status = metav1.ConditionTrue
```

Comment:
Can we also provide meaningful [Messages](https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/types.go#L742) for the condition

Reply:
added